testing/endpoint.go

```diff
@@ -566,11 +566,11 @@ func (endpoint *Endpoint) TimeoutOnClose(packet channeltypes.Packet) error {
 	return endpoint.Chain.sendMsgs(timeoutOnCloseMsg)
 }
 
-// SetChannelClosed sets a channel state to CLOSED.
-func (endpoint *Endpoint) SetChannelClosed() error {
+// SetChannelState sets a channel state
+func (endpoint *Endpoint) SetChannelState(state channeltypes.State) error {
```

Comment:
Nice! We could avoid the api breaking change here in the testing package by just adding a new function `SetChannelState(state)` and keep `SetChannelClosed` as well.

Reply:
I like this refactoring. Should we anyway break the API and document this in the migration docs for v7?

Reply:
I agree with @crodriguezvega that breaking the API and updating migration docs in this case is cleaner...do you have a strong opinion about not breaking it @damiannolan ?